docs/src/model_creation/model_file_loading_and_export.md

```diff
@@ -0,0 +1,163 @@
+```@example file_handling_2
+loaded_sol = deserialize("saved_parameters.jls")
+```
+
+## [Loading .net files using ReactionNetworkImporters.jl](@id file_loading_rni_net)
+A general-purpose format for storing CRN models is so-called .net files. These can be generated by e.g. [BioNetGen](https://bionetgen.org/). The [ReactionNetworkImporters.jl](https://github.com/SciML/ReactionNetworkImporters.jl) package enables the loading of such files to Catalyst `ReactionSystem`. Here we load a [Repressilator](@ref basic_CRN_library_repressilator) model stored in the "repressilator.net" file:
+```julia
+using ReactionNetworkImporters
+prn = loadrxnetwork(BNGNetwork(), "repressilator.net")
+```
+Here, .net files not only contain information regarding the reaction network itself, but also the numeric values (initial conditions and parameter values) required for simulating it. Hence, `loadrxnetwork` generates a `ParsedReactionNetwork` structure, containing all this information. You can access the model as `prn.rn`, the initial conditions as `prn.u0`, and the parameter values as `prn.p`. Furthermore, these initial conditions and parameter values are also made [*default* values](@ref dsl_advanced_options_default_vals) of the model. 
+
+A parsed reaction network's content can then be provided to various problem types for simulation. E.g. here we perform an ODE simulation of our repressilator model:
+```julia
+using Catalyst, OrdinaryDiffEq, Plots
+tspan = (0.0, 10000.0)
+oprob = ODEProblem(prn.rn, Float64[], tspan, Float64[])
+sol = solve(oprob)
+plot(sol; idxs = [:mTetR, :mLacI, :mCI])
+```
+![Repressilator Simulation](../assets/repressilator_sim_ReactionNetworkImporters.svg)
+
+Note that, as all initial conditions and parameters have default values, we can provide empty vectors for these into our `ODEProblem`.
+
+
+!!! note
+    It should be noted that .net files support a wide range of potential model features, not all of which are currently supported by ReactionNetworkImporters. Hence, there might be some .net files which `loadrxnetwork` will not be able to load.
+
+A more detailed description of ReactionNetworkImporter's features can be found in its [documentation](https://docs.sciml.ai/ReactionNetworkImporters/stable/).
+
+## Loading SBML files using SBMLImporter.jl and SBMLToolkit.jl
+The Systems Biology Markup Language (SBML) is the most widespread format for representing CRN models. Currently, there exist two different Julia packages, [SBMLImporter.jl](https://github.com/sebapersson/SBMLImporter.jl) and [SBMLToolkit.jl](https://github.com/SciML/SBMLToolkit.jl), that are able to load SBML files to Catalyst `ReactionSystem` structures. SBML is able to represent a *very* wide range of model features. Some of these are not supported by these packages (or Catalyst). Hence, there exist SBML files (typically containing obscure model features such as events with time delays) that currently cannot be loaded into Catalyst models).
+
```

Comment:
SBML is able to represent a *very* wide range of model features, and both packages support most features. However, there exist SBML files

As SBMLImporter supports a lot of weird features (more than most other SysBio tools), I think it should be highlighted here